src/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensSchema.cs

```diff
@@ -117,14 +107,21 @@ public SemanticTokensSchema(IReadOnlyDictionary<string, string> tokenTypeMap)
 
         public static SemanticTokensSchema GetSchema(bool clientSupportsVisualStudioExtensions)
             => clientSupportsVisualStudioExtensions
-                ? s_vsTokenSchema
+                ? LegacyTokensSchemaForLSIF
```

Comment:
Don't know if I particularly like this change.  I think it was more readable before - now it seems weird that that its using the LegacyTokensSchemaForLsif

Reply:
ok.  i'll revert that one.